source/process/docs-contributors-guide.rst

```diff
@@ -0,0 +1,35 @@
+Contributor's Guide for docs.mattermost.com
+===========================================
+
+Thank you for your interest in contributing to Mattermost documentation. Your contributions benefit thousands of organizations around the world deploying Mattermost. 
+
+To start: 
+
+Create a Ticket 
+---------------
+
+1. **Select a Help Wanted ticket** - Please select a `HELP WANTED ticket <https://github.com/mattermost/docs/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22>`_ that you'd like to address with your contribution, or create a new ticket to discuss an improvement you'd like to make and wait for a Project Member to approve and add the `Help Wanted` label so the change will be accepted. Project Members have a "Member" label on their comments. 
+
+If your update is small and an obvious improvement, you can skip this step and submit a PR directly. 
+
+2. **Become an Approved Contributor** - Please complete the `contributor license agreement <https://www.mattermost.org/mattermost-contributor-agreement/>`_ (a.k.a. "CLA"). This is a standard form for most open source projects and it adds your the `approved contributor's list <https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true>`_ for the Mattermost open source project. There are answers to frequently asked questions in the link above. 
+
+3. **Create your Pull Request** - Please create a pull request for the documentation change you would like to make. Make sure to review and follow the `Style Guide XXX <https://docs.mattermost.com/process/sg_mattermost-doc-style.html>`_. Please include a link to the Help Wanted ticket if you have one. 
+
+4. **(Optional) Meet the Documentation Community** - You can join the `Mattermost Documentation channel <https://pre-release.mattermost.com/core/channels/documentation>`_ on the community server and introduce yourself to other contributors and project members. 
+
+Once your ticket is submitted it goes into a review process: 
+
+PR Review Process for Contributors 
+----------------------------------
+
+1. **New PR submitted** - Each PR created sends a notification to the `Documentation Channel <https://pre-release.mattermost.com/core/channels/documentation>`_ on the Mattermost contributors server, alerting reviewers. 
+
+2. **Triage** - Project Member assigned to triage reviews doc repo within 1-2 working days and applies the `appropriate "Needs ... Approval" labels to indicate required approvals XXX <XXX>`_. Use **Saved Reply** template linked to this page to let submitter know what to expect.
+
+3. **Reviews** - Within 5-10 working days, for each `"Needs ... Approval" XXX <link to labels explaination>`_ label, reviewers on rotation either a) leave feedback or b) mark PR "Approved" and remove the appropriate label. The reviewer requiring changes adds "Awaiting Submitter Action" label if it is not yet applied. After submitter addresses feedback, the 5-10 working day clock restarts.
+
```

Comment:
I think in practice, "5-10 working days" will end up being "10 working days", which seems kind of long.